src/olympia/addons/serializers.py

```diff
@@ -497,7 +514,7 @@ class ESAddonSerializer(BaseESSerializer, AddonSerializer):
     # data the same way than the regular serializer does (usually because we
     # some of the data is not indexed in ES).
     authors = BaseUserSerializer(many=True, source='listed_authors')
-    current_version = SimpleESVersionSerializer()
```

Comment:
I removed it... but had to re-introduce it with another name because I needed a version serializer that inherits from `BaseESSerializer` for the translations to work.